apps/meteor/tests/e2e/admin-users-custom-fields.spec.ts

```diff
@@ -0,0 +1,139 @@
+			await poAdmin.inputSearchUsers.fill(addTestUser.data.username);
+
+			await expect(poAdmin.getUserRowByUsername(addTestUser.data.username)).toBeVisible();
+			await poAdmin.getUserRowByUsername(addTestUser.data.username).click();
+		});
+
+		await test.step('should navigate to edit user form', async () => {
+			await poAdmin.btnEdit.waitFor();
+			await poAdmin.btnEdit.click();
+		});
+
+		await test.step('should fill custom fields for user', async () => {
+			await poAdmin.tabs.users.inputName.waitFor();
+
+			await expect(poAdmin.tabs.users.getCustomField('customFieldText1')).toBeVisible();
+			await expect(poAdmin.tabs.users.getCustomField('customFieldText2')).toBeVisible();
+
+			await poAdmin.tabs.users.getCustomField('customFieldText1').fill(adminCustomFieldValue1);
+			await poAdmin.tabs.users.getCustomField('customFieldText2').fill(adminCustomFieldValue2);
+		});
+
+		await test.step('should save user custom fields', async () => {
+			await poAdmin.tabs.users.btnSaveUser.waitFor();
+			await poAdmin.tabs.users.btnSaveUser.click();
+			await expect(poHomeChannel.toastSuccess).toBeVisible();
+			await poHomeChannel.dismissToast();
+		});
+
+		await test.step('should verify custom fields were saved', async () => {
+			await poAdmin.tabs.users.btnContextualbarClose.click();
+			await poAdmin.getUserRowByUsername(addTestUser.data.username).click();
+			await poAdmin.btnEdit.click();
+
+			await expect(poAdmin.tabs.users.getCustomField('customFieldText1')).toHaveValue(adminCustomFieldValue1);
+			await expect(poAdmin.tabs.users.getCustomField('customFieldText2')).toHaveValue(adminCustomFieldValue2);
+		});
+	});
+
+	test('should allow admin to update existing user custom fields', async () => {
+		await test.step('should find and click on update test user', async () => {
+			await poAdmin.inputSearchUsers.fill(updateTestUser.data.username);
+
+			await expect(poAdmin.getUserRowByUsername(updateTestUser.data.username)).toBeVisible();
+			await poAdmin.getUserRowByUsername(updateTestUser.data.username).click();
+		});
+
+		await test.step('should navigate to edit user form', async () => {
+			await poAdmin.btnEdit.waitFor();
+			await poAdmin.btnEdit.click();
+		});
+
+		await test.step('should verify existing values and update one custom field', async () => {
+			await poAdmin.tabs.users.inputName.waitFor();
+
+			await expect(poAdmin.tabs.users.getCustomField('customFieldText1')).toHaveValue(customFieldInitial1);
+			await expect(poAdmin.tabs.users.getCustomField('customFieldText2')).toHaveValue(adminCustomFieldValue2);
+
+			await poAdmin.tabs.users.getCustomField('customFieldText1').clear();
+			await poAdmin.tabs.users.getCustomField('customFieldText1').fill(adminCustomFieldUpdated1);
+		});
+
+		await test.step('should save and verify partial update', async () => {
+			await poAdmin.tabs.users.btnSaveUser.waitFor();
+			await poAdmin.tabs.users.btnSaveUser.click();
+			await expect(poHomeChannel.toastSuccess).toBeVisible();
```

Comment:
IMO we should avoid this, unfortunately something the team discourage but it's not written anywhere